pkg/ssh/client/ssh.go

```diff
@@ -133,8 +133,11 @@ func importKey(ctx context.Context, keyName, fingerprint string, keyContent *str
 	if existing, err := findKeyInEC2(ctx, ec2API, keyName); err != nil {
 		return err
 	} else if existing != nil {
-		if *existing.KeyFingerprint != fingerprint {
-			return fmt.Errorf("SSH public key %s already exists, but fingerprints don't match (exected: %q, got: %q)", keyName, fingerprint, *existing.KeyFingerprint)
+		// AWS adds an extra `=` to ed25519 sha256 encodings. Normally, that's not there.
+		// Because it's just base64 padding it's fine to remove and compare without it.
+		trimmedAWSFingerprint := strings.TrimSuffix(*existing.KeyFingerprint, "=")
```

Comment:
Is the padding always going to be up to 1 `=`, not more?

Reply:
Yep :nod: